### PR TITLE
code refactor

### PR DESCRIPTION
**Pull request by domino644**
changes all variable names to be lower case

### CODE REVIEW

File: src/tools/fat_reader.c [hunk 1]
```diff
@@ -6,178 +6,186 @@ typedef uint8_t bool;
...
+    uint16_t firstLogicalCluster;
+    uint32_t fileSize;
+
+} __attribute__((packed)) DirEntry;
 
 BootSector disk_BootSector;
-uint8_t* FATTable = NULL;
-DirEntry* disk_RootEntry = NULL;
+uint8_t *FAT_Table = NULL;
+DirEntry *disk_RootEntry = NULL;
```
Comment (Interphaase) [inline src/tools/fat_reader.c]: Star after variable type declaration
(convention)

**Review by Interphaase — COMMENTED** [review]
camelCase ok, change pointer declarations to match the convention

File: src/tools/fat_reader.c (hunk 1 above)
Comment (domino644) [reply src/tools/fat_reader.c]: can we stay to type *name convenion because my autoformatter in vsc automatically does it?

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 ./build/*
 ./bochs_config
+.editorconfig
```
Comment (Interphaase) [inline .gitignore:3]: Remove it, only you are using editorconfig

**Review by Interphaase — COMMENTED** [review]
Remove editor config from .gitignore

**Pull request merged**